index.ts

```diff
@@ -0,0 +1,212 @@
+// ===== Type definitions =====
+type Place = {
+    name: string;
+    lat: number;
+    lon: number;
+}
+
+type WeatherData = {
+    airTemp: number;
+    condition: number;
+    validTime: string;
+}
+
+type DailyForecast = {
+    date: string;
+    minTemp: number;
+    maxTemp: number;
+    condition: number;
+}
+
+// ===== List of cities =====
+const places: Place[] = [
+    { name: "Stockholm", lat: 59.329468, lon: 18.062639 },
+    { name: "Uppsala", lat: 59.8586, lon: 17.6389 },
+    { name: "Gotland", lat: 57.636, lon: 18.294 },
+    { name: "Umeå", lat: 63.8258, lon: 20.263 },
```

Comment:
good solution to store some cities like this. maybe the background image could be put in here as another property instead of having that in a separate object?